app/src/main/java/com/zulip/android/util/Constants.java

```diff
@@ -13,5 +13,6 @@
     //Default maximum time limit for editing message(Same as server)
     public final static int DEFAULT_MAXIMUM_CONTENT_EDIT_LIMIT = 600;
     public final static boolean DEFAULT_EDITING_ALLOWED = true;
+    public static String serverUrl = null;
```

Comment:
We are using `Constant.java` for all constant used in the project (all are final, no need to set at the runtime).
I think it will be better to use `AppPreferences` for storing `serverUrl`